Aleph1.Skeletons.WebAPI/Aleph1.Skeletons.WebAPI.Pages/index.ejs

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
-<html>
+<html lang="he">
 
-	<head>
-		<meta charset="utf-8">
-		<title>Demo App</title>
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<base href="<%- htmlWebpackPlugin.options.metadata.baseUrl %>">
-		<link rel="icon" href="favicon.ico">
+<head>
+	<meta charset="utf-8">
+	<title>Demo App</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<base href="<%- htmlWebpackPlugin.options.metadata.baseUrl %>">
+	<link rel="icon" href="favicon.ico">
+</head>
 
-		<!-- imported CSS are concatenated and added automatically -->
-	</head>
+<body>
+	<div id="app" aurelia-app="main"></div>
```

Comment:
See:
https://github.com/726b/Aleph1.Skeletons/blob/0102c01b53d4c62f952cc727df29128aea581e21/Aleph1.Skeletons.WebAPI/Aleph1.Skeletons.WebAPI.Pages/src/styles/base.scss#L33-L36

Using ID instead of class because it is a unique element that appears only once and is never changed.